src/Microsoft.AspNetCore.BuildTools.ApiCheck/Microsoft.AspNetCore.BuildTools.ApiCheck.csproj

```diff
@@ -21,4 +21,29 @@
     <PackageReference Include="System.Runtime.Loader" Version="4.0.0" />
   </ItemGroup>
 
+  <!-- packaging settings-->
+  <PropertyGroup>
+    <NoPackageAnalysis>true</NoPackageAnalysis>
+    <NuspecFile>$(MSBuildThisFileDirectory)$(MSBuildProjectName).nuspec</NuspecFile>
+    <IntermediatePackDir>$(MSBuildThisFileDirectory)bin\$(Configuration)\publish\</IntermediatePackDir>
+    <PublishDir>$(IntermediatePackDir)$(TargetFramework)\</PublishDir>
+    <NuspecBasePath>$(IntermediatePackDir)</NuspecBasePath>
+  </PropertyGroup>
+
+  <Target Name="PublishAll">
+    <ItemGroup>
+      <_TargetFrameworks Remove="@(_TargetFrameworks)"/>
```

Comment:
Clear the itemgroup and does a cross-targeting build. The "Publish" task requires you set a TargetFramework.